src/time-picker/time-picker.scss

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-time-picker {
+  @include iui-reset;
+  @include iui-font-family;
+  user-select: none;
+  text-align: center;
+  height: $iui-baseline * 26;
+  display: inline-flex;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+
+  &:not(:first-child) {
+    @include themed {
+      border-left: 1px solid t(iui-color-background-4);
+    }
+  }
+
+  &:first-child {
+    box-shadow: $iui-elevation-2;
+  }
+
+  > .iui-time,
+  > .iui-period {
+    padding: $iui-baseline $iui-s;
+    overflow-y: auto;
+
+    &:not(:first-child) {
```

Comment:
Does it need to be repeated?

Reply:
well yes.. Unless you can find a better way, how to handle different situations (time picker alone, time picker in date picker, etc...)